fastlane/Fastfile

```diff
@@ -0,0 +1,34 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+
+platform :ios do
+  desc "Deploy a new beta build to iTunes Connect"
+  lane :deploy do
+    upload_to_testflight(
+      ipa: "status/StatusIm.ipa",
+      changelog: "Improvements and bug fixes."
+    )
+    slack(message: "New build uploaded to iTunes Connect")
+  end
+end
+
+
+platform :android do
+  desc "Deploy a new alpha build to Google Play"
+  lane :deploy do
+    upload_to_play_store(
+      track: "alpha",
+      apk: "android/app/build/outputs/apk/release/app-release.apk"
+    )
+    slack(message: "New build uploaded to Google Play")
+  end
+end
```

Comment:
Have you thought about adding
```
lane :clean do
  clean_testflight_testers(dry_run: true, days_of_inactivity: 60)
  slack(message: "Cleaned TestFlight users")
end
```
Or something similar here? Not necessary, but nice for completion.

Reply:
That's a nice touch. There is so much more I wanted to add but thought it best for a future change.